src/config/configManager.go

```diff
@@ -176,6 +176,19 @@ func LoadConfigFromFile() {
 		FileFromSource:    true,
 	}
 
+	var sys_ns, sys_notNs []string
+	sys_namespaces := viper.GetStringSlice("application.system.namespace-filter")
+	for _, n := range sys_namespaces {
+		if n[0] == '!' {
+			sys_notNs = append(sys_notNs, n[1:])
+		} else {
+			sys_ns = append(sys_ns, n)
+		}
+	}
```

Comment:
Can we refactor the code ... between application and network? Seems we can create a function to read the filter and return the ns and notNs vars.